test/verify/check-kdump

```diff
@@ -32,12 +32,7 @@ class TestKdump(MachineCase):
 
     def enableKdump(self):
         # we need to make sure that the kernel command line options include our crashkernel parameter
-        if self.machine.atomic_image:
-            # on atomic we have a crashkernel option already, but for auto
-            contents = self.machine.execute(command="cat /boot/grub2/grub.cfg", quiet=True)
-            self.assertIn("crashkernel=auto", contents)
-            self.machine.write("/boot/grub2/grub.cfg", contents.replace("crashkernel=auto", "crashkernel=256M"))
-        elif self.machine.image in ["rhel-8-0", "rhel-8-0-distropkg", "rhel-8-1"]:
+        if self.machine.image in ["rhel-8-0", "rhel-8-0-distropkg", "rhel-8-1"]:
```

Comment:
I could have cleaned this up also.. I'll do that if I forgot something else, otherwise :shrug:

Reply:
Done.